### PR TITLE
clarify Trezor xpub solutions

### PR DESCRIPTION
**Pull request by JayOceans**

### CODE REVIEW

File: _pages/known-issues/hardware/trezor.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ title: Trezor
 Default usage leaks your xpub to Trezor's servers (and whoever they share records with).
 This is [estimated at 99% of their users](https://coldbit.com/how-to-use-a-trezor-without-leaking-your-balance-and-transaction-history-to-third-parties/).
 This has severe privacy implications and some [potential security implications](https://www.reddit.com/r/Bitcoin/comments/5csles/what_are_the_security_risks_of_sharing_xpub/).
+A simple solution is to initialize your Trezor with the Trezor Manager app (android) & use it with Specter/Core.
```
Comment (mflaxman) [inline _pages/known-issues/hardware/trezor.md:11]: Can you include a link with explanation for this? I've never heard of it, and obviously it has risks as well that need instructions (I assume you'd want to disable wifi).

File: _pages/known-issues/hardware/trezor.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ title: Trezor
...
+The CLI solution, linked above, enables more options.
```
Comment (mflaxman) [inline _pages/known-issues/hardware/trezor.md:12]: Can you clarify this? There's no mention of CLI solutions on the page.

Comment (JayOceans) [reply _pages/known-issues/hardware/trezor.md:12]: ya, that's not super clear...

the article by Coldbit, linked above, shows how to use Trezor in CLI... 

// i know this is a guide for multi-sig, but why not support best practices for single-sig users, while we're at it...

File: _pages/known-issues/hardware/trezor.md (hunk 1 above)
Comment (JayOceans) [reply _pages/known-issues/hardware/trezor.md:11]: this has been around for awhile
but only useful to me now that
i can use Trezor with Specter/ Core
and not only Trezor's web wallet...
https://blog.trezor.io/trezor-manager-app-for-android-5f8b86bfc886

File: _pages/known-issues/hardware/trezor.md (hunk 2 above)
Comment (mflaxman) [reply _pages/known-issues/hardware/trezor.md:12]: I don't believe there are best practices for single sig, it's too hard for all but experts to do safely!

**Comment by mflaxman** [comment]
I'm happy to link to ways to protect your privacy with the android app, but there's no explanation of how to do it in the link your commented.

Closing this for now as PRs are for pull requests, not issue discussions (feel free to open an issue).

Thanks!

**Pull request closed without merging**